src/FulfillerMultiSelect.jsx

```diff
@@ -119,20 +148,53 @@ class FulfillerMultiSelect extends React.Component {
     async getRecentFulfillerIds() {
         if (this.props.accessToken) {
             let settings = await this.customizrClient.getSettings(this.props.accessToken);
-            let recentFulfillerIds = settings.recentFulfillerIds || [];
-            this.setState({ recentFulfillerIds });
-            return recentFulfillerIds;
+            return {
+                recentFulfillerIds: settings.recentFulfillerIds || [],
+                recentFulfillerIdsMulti: settings.recentFulfillerIdsMulti || []
+            };
+        }
+    }
+
+    recentFulfillersChanged(fulfillerIds) {
+        const recentFulfillerIds = this.state.recentFulfillerIds;
+
+        if (recentFulfillerIds.length === 0) {
+            return false;
+        }
+
+        if (fulfillerIds > recentFulfillerIds) {
+            return true;
+        }
+
+        if (this.props.multi) {
+            return fulfillerIds.length !== recentFulfillerIds.length
+                || fulfillerIds.any((fulfillerId, i) => recentFulfillerIds[i] === fulfillerId);
```

Comment:
This condition should be tweaked. The `any` part doesn't make much sense (Customizr will get updated as long as the first entry in the fulfiller ID list is the same).

Reply:
It might be that this condition is not needed at all. Apparently the only action possible to be taken by the user are:
- add a fulfiller
- remove a fulfiller.

It is not possible to add a fulfiller which is already part of the selection, either.
Therefore, it may be possible to get rid of the condition altogether.
This might allow the entire function to be rewritten as: `fulfillerIds.length !== recentFulfillerIds.length && fulfillerIds.length !== 0`. (I haven't checked all the cases).